src/senso.js

```diff
@@ -40,7 +40,20 @@ var interfaces = R.filter(R.identity, R.flatten(R.values(R.map(R.map((ipInfo) =>
   }
 }), os.networkInterfaces()))))
 
-const bonjour = require('bonjour')({interface: interfaces})
+const bonjour = require('bonjour')({interface: interfaces, reuseAddr: true})
+
+// Handle case where binding to interface fails. NOTE: this is not standard multicast-dns API!
```

Comment:
This sets `SO_REUSEADDR` (or `SO_REUSEPORT`) which allows coexistance of system mDNS service and our mDNS client in certain situations (notably avahi on Linux).

On Mac coexistance with the system mDNS service does not work. I assume because the system service does not set `SO_REUSEPORT`:

> SO_REUSEPORT is what most people would expect SO_REUSEADDR to be. Basically, SO_REUSEPORT allows you to bind an arbitrary number of sockets to exactly the same source address and port as long as all prior bound sockets also had SO_REUSEPORT set before they were bound.
(http://stackoverflow.com/questions/14388706/socket-options-so-reuseaddr-and-so-reuseport-how-do-they-differ-do-they-mean-t)